src/igvfd/audit/tissue.py

```diff
@@ -18,3 +18,15 @@ def audit_tissue_ccf_id(value, system):
             f'is missing common coordinate framework identifier (CCF ID) required for human data.'
         )
         yield AuditFailure('missing ccf_id', detail, level='NOT_COMPLIANT')
+
+
+@audit_checker('Tissue', frame='object')
+def audit_tissue_ccf_id_nonhuman_sample(value, system):
+    '''Tissue objects must not specify a common coordinate framework identifier (CCF ID) unless the sample is from a human donor.'''
+    if ('ccf_id' in value) and (value.get('taxa') != 'Homo sapiens'):
```

Comment:
if it is a mixed donors sample you might not have taxa. I think this should be (value.get('taxa') -> (value.get('taxa', '')